src/components/Graph.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useRef } from 'react';
+import { css } from 'emotion';
+import { useEffect } from 'react';
+import { draw } from '../helpers/Graph/Draw';
+import * as d3 from 'd3';
+import { DependencyLink, DependencyNode } from './types';
+
+export interface GraphProps {
+    nodes: DependencyNode[];
+    links: DependencyLink[];
+}
+
+export const Graph = React.memo<GraphProps>(({ nodes, links }) => {
```

Comment:
GraphProps is the same as Network interface
write: ``React.memo<Network>``

Reply:
Graph component is changing quickly, don't want to be dependent on Network interface.